contracts/ERC20Interface.sol

```diff
@@ -10,5 +10,6 @@ interface ERC20 {
     function approve(address _spender, uint _value) external returns (bool success);
     function allowance(address _owner, address _spender) external view returns (uint remaining);
     function decimals() external view returns(uint digits);
+    function symbol() external view returns(string);
```

Comment:
please don't change production code for this feature.
make another interface in the wrapper if needed.